food-truck-api/src/main/java/food/truck/api/truck/TruckRepository.java

```diff
@@ -9,4 +9,5 @@
 public interface TruckRepository extends JpaRepository<Truck, Long> {
     List<Truck> findByName(String name);
     List<Truck> findByUserId(Long userId);
+    List<Truck> findByNameLikeOrDescriptionLike(String name, String description);
```

Comment:
I know there's an 'IgnoreCase' keyword you could apply to add case-insensitivity to the matching, but I'm not sure how to use it.